frontend/.angular-cli.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
   "project": {
-    "version": "1.0.0-beta.32.3",
+    "version": "1.0.0-rc.0",
```

Comment:
no need for the version number in this file anymore with rc0 (yep, I read the changelog this morning :P)